custom_components/better_thermostat/model_fixes/SEA801-Zigbee_SEA802-Zigbee.py

```diff
@@ -0,0 +1,17 @@
+def fix_local_calibration(self, entity_id, offset):
+    # device SEA802 fix
+    if offset > -1.5:
```

Comment:
we need here another condition, if it shall actually heat, otherwise it's always set -1.5

Reply:
Yes, i changed it